docs/predictive-analytics/model-retraining/process.sql

```diff
@@ -0,0 +1,64 @@
+SET 'auto.offset.reset' = 'earliest';
+
+-- Create stream of predictions
+CREATE STREAM PREDICTED_WEIGHT(
+  "Fish_Id" VARCHAR KEY,
+  "Species" VARCHAR,
+  "Height" DOUBLE,
+  "Length" DOUBLE,
+  "Prediction" DOUBLE
+  )
+WITH(
+  KAFKA_TOPIC = 'kt.mdb.weight-prediction', 
+  VALUE_FORMAT = 'JSON'
+);
+
+-- Create stream of actual weights
+CREATE STREAM ACTUAL_WEIGHT(
+  "Fish_Id" VARCHAR KEY,
+  "Species" VARCHAR,
+  "Weight" DOUBLE
+  )
+WITH(
+  KAFKA_TOPIC = 'kt.mdb.machine-weight', 
+  VALUE_FORMAT = 'JSON'
+);
+
+-- Create stream joining predictions with actual weights
+CREATE STREAM DIFF_WEIGHT
+WITH(
+  KAFKA_TOPIC = 'weight-diff', 
```

Comment:
nit: consistent naming: topic and stream name ordering of `weight` and `diff`

Reply:
I had left this inconsistency because it was this way in the blog post, but I am changing some other things in the code to simplify it, so I changed this too.